scripts/pre_commit_linter.py

```diff
@@ -57,6 +58,10 @@
     '--path',
     help='path to the directory with files to be linted',
     action='store')
+_PARSER.add_argument(
+    '--silent',
```

Comment:
As per conversation with @seanlip, this option will be eliminated in favor of a print out a status every 10 files it is linting